tests/snapshots/files__gamma_condition_and-optimize.snap

```diff
@@ -14,21 +14,13 @@ expression: visualization.result
   br v3_ .b8_ .b9_;
 .b8_:
   c10_: int = const 3;
-  v11_: int = id c10_;
-  br v4_ .b12_ .b13_;
-.b12_:
-  v11_: int = id c5_;
+  v11_: int = select v4_ c5_ c10_;
```

Comment:
Nice that the `select` optimization is applied. Any idea why it was not applied before?

Reply:
It's flaky, probably an extraction problem or same cost